src/modules/extendedCallList/i18n/fr_FR.root.json

```diff
@@ -172,12 +172,12 @@
             "title": "Marquer les missions saisonnières"
         },
         "hide0CurrentPatients": {
-            "description": "Masque le nombre de patients s'il n'y en a pas sur place",
+            "description": "Masquer le nombre de patients s'il n'y en a pas sur place",
             "title": "Masquer 0 patient"
         },
         "hide0CurrentPrisoners": {
-            "description": "Hides amount of patients if there are no prisoners at scene",
-            "title": "Hide 0 Prisoners"
+            "description": "Masquer le nombre de prisonniers s'il n'y en a pas sur place",
+            "title": "HMasquer 0 Prisonniers"
```

Comment:
I think there a leftover `H` at the beginning.

Reply:
thank :)